packages/rum/src/domain/requestCollection.ts

```diff
@@ -48,14 +44,10 @@ export function startRequestCollection(lifeCycle: LifeCycle, configuration: Conf
 }
 
 export function trackXhr(lifeCycle: LifeCycle, configuration: Configuration, tracer: Tracer) {
-  const xhrProxy = startXhrProxy<CustomContext & XhrStartContext, CustomContext & XhrCompleteContext>()
+  const xhrProxy = startXhrProxy<CustomContext & TracedXhrStartContext, CustomContext & TracedXhrCompleteContext>()
```

Comment:
`CustomContext & XXXContext` had the advantage of make it clear what was added to the context in this file.
IMO, with only `requestId` in `CustomContext`, `CustomContext` does not really make sense anymore.
What about merging it with `TracedXXXContext`?

Reply:
I agree, I will move it.